packages/text-field/helper-text/README.md

```diff
@@ -30,6 +30,39 @@ role | String | Sets role on HTML element.
 showToScreenReader | Boolean | toggles the aria-hidden property to allow element to be visible to the screen reader.
 validation | Boolean | If true, alters the helper text to an error message.
 
+## Input Validation
+
+HelperText provides validity styling by setting the `validation` prop in HelperText. Validation can be checked through the appropriate `Input` validation properties (pattern, min, max, require, step, minlength, maxlength).
+
+The following snipped is an example of how to use regex and check for minimum length with HelperText:
```

Comment:
- snipped --> snippet
- regex --> pattern regex

Reply:
How did the `d` get in there, it's so far apart from the `t`.
I'll change that too.